File: app/buildz/tmp/kapt3/stubs/debug/com/bits/bee/bpmc/data/data_source/remote/ApiUtils.java
package com.bits.bee.bpmc.data.data_source.remote;

import java.lang.System;

/**
 * Created by aldi on 25/02/22.
 */
@kotlin.Metadata(mv = {1, 7, 1}, k = 1, d1 = {"\u0000\u00e4\u0001\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0000\n\u0002\u0018\u0002\n\u0002\b\u0002\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\u0018\u00002\u00020\u0001B\u000f\b\u0007\u0012\u0006\u0010\u0002\u001a\u00020\u0003\u00a2\u0006\u0002\u0010\u0004J\u0006\u0010\u0005\u001a\u00020\u0006J\u0006\u0010\u0007\u001a\u00020\bJ\u0006\u0010\t\u001a\u00020\nJ\u0006\u0010\u000b\u001a\u00020\fJ\u0006\u0010\r\u001a\u00020\u000eJ\u0006\u0010\u000f\u001a\u00020\u0010J\u0006\u0010\u0011\u001a\u00020\u0012J\u0006\u0010\u0013\u001a\u00020\u0014J\u0006\u0010\u0015\u001a\u00020\u0016J\u0006\u0010\u0017\u001a\u00020\u0018J\u0006\u0010\u0019\u001a\u00020\u001aJ\u0006\u0010\u001b\u001a\u00020\u001cJ\u0006\u0010\u001d\u001a\u00020\u001eJ\u0006\u0010\u001f\u001a\u00020 J\u0006\u0010!\u001a\u00020\"J\u0006\u0010#\u001a\u00020$J\u0006\u0010%\u001a\u00020&J\u0006\u0010\'\u001a\u00020(J\u0006\u0010)\u001a\u00020*J\u0006\u0010+\u001a\u00020,J\u0006\u0010-\u001a\u00020.J\u0006\u0010/\u001a\u000200J\u0006\u00101\u001a\u000202J\u0006\u00103\u001a\u000204J\u0006\u00105\u001a\u000206J\u0006\u00107\u001a\u000208J\u0006\u00109\u001a\u00020:J\u0006\u0010;\u001a\u00020<J\u0006\u0010=\u001a\u00020>J\u0006\u0010?\u001a\u00020@J\u0006\u0010A\u001a\u00020BJ\u0006\u0010C\u001a\u00020DJ\u0006\u0010E\u001a\u00020FJ\u0006\u0010G\u001a\u00020HJ\u0006\u0010I\u001a\u00020JR\u000e\u0010\u0002\u001a\u00020\u0003X\u0082\u0004\u00a2\u0006\u0002\n\u0000\u00a8\u0006K"}, d2 = {"Lcom/bits/bee/bpmc/data/data_source/remote/ApiUtils;", "", "retrofitClient", "Lcom/bits/bee/bpmc/data/data_source/remote/RetrofitClient;", "(Lcom/bits/bee/bpmc/data/data_source/remote/RetrofitClient;)V", "getAddOnApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/AddOnApiService;", "getAddOnDApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/AddOnDApiService;", "getAuthApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/AuthApiService;", "getBpApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/BpApiService;", "getBranchApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/BranchApiService;", "getCashierApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/CashierApiService;", "getCcTypeApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/CcTypeApiService;", "getChannelApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ChannelApiService;", "getCityApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/CityApiService;", "getCmpApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/CmpApiService;", "getCrcApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/CrcApiService;", "getDistrictApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/DistrictApiService;", "getEdcApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/EdcApiService;", "getEdcSurcApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/EdcSurcApiService;", "getGrpPrvApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/GrpPrvApiService;", "getInitialApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/InitialApiService;", "getItemAddOnApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ItemAddOnApiService;", "getItemApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ItemApiService;", "getItemGroupApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ItemGroupApiService;", "getItemVariantApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ItemVariantApiService;", "getPmtdApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/PmtdApiService;", "getPriceApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/PriceApiService;", "getPriceLvlApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/PriceLvlApiService;", "getPromoApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/PromoApiService;", "getProvinceApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/ProvinceApiService;", "getRegApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/RegApiService;", "getRegencyApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/RegencyApiService;", "getSelectionApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/SelectionApiService;", "getSelectionDApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/SelectionDApiService;", "getSignUpApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/SignUpApiService;", "getStockApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/StockApiService;", "getTaxApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/TaxApiService;", "getUnitApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/UnitApiService;", "getUsrGrpApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/UsrGrpApiService;", "getVariantApiService", "Lcom/bits/bee/bpmc/data/data_source/remote/apiservices/VariantApiService;", "app_debug"})
public final class ApiUtils {
    private final com.bits.bee.bpmc.data.data_source.remote.RetrofitClient retrofitClient = null;
    
    @javax.inject.Inject()
    public ApiUtils(@org.jetbrains.annotations.NotNull()
    com.bits.bee.bpmc.data.data_source.remote.RetrofitClient retrofitClient) {
        super();
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.AuthApiService getAuthApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.BranchApiService getBranchApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.CashierApiService getCashierApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.CityApiService getCityApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.BpApiService getBpApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.TaxApiService getTaxApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.PmtdApiService getPmtdApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ChannelApiService getChannelApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ItemGroupApiService getItemGroupApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ItemApiService getItemApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.PriceLvlApiService getPriceLvlApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.CrcApiService getCrcApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.CmpApiService getCmpApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.AddOnApiService getAddOnApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.AddOnDApiService getAddOnDApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ItemAddOnApiService getItemAddOnApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.VariantApiService getVariantApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ItemVariantApiService getItemVariantApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.SelectionApiService getSelectionApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.SelectionDApiService getSelectionDApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.RegApiService getRegApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.PromoApiService getPromoApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.GrpPrvApiService getGrpPrvApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.UsrGrpApiService getUsrGrpApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.CcTypeApiService getCcTypeApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.EdcApiService getEdcApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.EdcSurcApiService getEdcSurcApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.ProvinceApiService getProvinceApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.RegencyApiService getRegencyApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.DistrictApiService getDistrictApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.InitialApiService getInitialApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.PriceApiService getPriceApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.StockApiService getStockApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.UnitApiService getUnitApiService() {
        return null;
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.bits.bee.bpmc.data.data_source.remote.apiservices.SignUpApiService getSignUpApiService() {
        return null;
    }
}

File: app/src/main/java/com/bits/bee/bpmc/data/data_source/remote/ApiUtils.kt
package com.bits.bee.bpmc.data.data_source.remote

import com.bits.bee.bpmc.data.data_source.remote.apiservices.*
import com.bits.bee.bpmc.utils.BPMConstants
import javax.inject.Inject

/**
 * Created by aldi on 25/02/22.
 */

class ApiUtils @Inject constructor(private val retrofitClient: RetrofitClient) {

    fun getAuthApiService() : AuthApiService {
        return retrofitClient.getClientProvision()!!.create(AuthApiService::class.java)
    }

    fun getBranchApiService() : BranchApiService {
        return retrofitClient.getClientApi()!!.create(BranchApiService::class.java)
    }

    fun getCashierApiService() : CashierApiService {
        return retrofitClient.getClientApi()!!.create(CashierApiService::class.java)
    }

    fun getCityApiService() : CityApiService {
        return retrofitClient.getClientApi()!!.create(CityApiService::class.java)
    }

    fun getBpApiService() : BpApiService {
        return retrofitClient.getClientApi()!!.create(BpApiService::class.java)
    }

    fun getTaxApiService() : TaxApiService {
        return retrofitClient.getClientApi()!!.create(TaxApiService::class.java)
    }

    fun getPmtdApiService() : PmtdApiService {
        return retrofitClient.getClientApi()!!.create(PmtdApiService::class.java)
    }

    fun getChannelApiService() : ChannelApiService {
        return retrofitClient.getClientApi()!!.create(ChannelApiService::class.java)
    }

    fun getItemGroupApiService() : ItemGroupApiService {
        return retrofitClient.getClientApi()!!.create(ItemGroupApiService::class.java)
    }

    fun getItemApiService() : ItemApiService {
        return retrofitClient.getClientApi()!!.create(ItemApiService::class.java)
    }

    fun getPriceLvlApiService() : PriceLvlApiService {
        return retrofitClient.getClientApi()!!.create(PriceLvlApiService::class.java)
    }

    fun getCrcApiService() : CrcApiService {
        return retrofitClient.getClientApi()!!.create(CrcApiService::class.java)
    }

    fun getCmpApiService() : CmpApiService {
        return retrofitClient.getClientApi()!!.create(CmpApiService::class.java)
    }

    fun getAddOnApiService() : AddOnApiService {
        return retrofitClient.getClientApi()!!.create(AddOnApiService::class.java)
    }

    fun getAddOnDApiService() : AddOnDApiService {
        return retrofitClient.getClientApi()!!.create(AddOnDApiService::class.java)
    }

    fun getItemAddOnApiService() : ItemAddOnApiService {
        return retrofitClient.getClientApi()!!.create(ItemAddOnApiService::class.java)
    }

    fun getVariantApiService() : VariantApiService {
        return retrofitClient.getClientApi()!!.create(VariantApiService::class.java)
    }

    fun getItemVariantApiService() : ItemVariantApiService {
        return retrofitClient.getClientApi()!!.create(ItemVariantApiService::class.java)
    }

    fun getSelectionApiService() : SelectionApiService {
        return retrofitClient.getClientApi()!!.create(SelectionApiService::class.java)
    }

    fun getSelectionDApiService() : SelectionDApiService {
        return retrofitClient.getClientApi()!!.create(SelectionDApiService::class.java)
    }

    fun getRegApiService() : RegApiService {
        return retrofitClient.getClientApi()!!.create(RegApiService::class.java)
    }

    fun getPromoApiService() : PromoApiService {
        return retrofitClient.getClientApi()!!.create(PromoApiService::class.java)
    }

    fun getGrpPrvApiService() : GrpPrvApiService {
        return retrofitClient.getClientApi()!!.create(GrpPrvApiService::class.java)
    }

    fun getUsrGrpApiService() : UsrGrpApiService {
        return retrofitClient.getClientApi()!!.create(UsrGrpApiService::class.java)
    }

    fun getCcTypeApiService() : CcTypeApiService {
        return retrofitClient.getClientApi()!!.create(CcTypeApiService::class.java)
    }

    fun getEdcApiService() : EdcApiService {
        return retrofitClient.getClientApi()!!.create(EdcApiService::class.java)
    }

    fun getEdcSurcApiService() : EdcSurcApiService {
        return retrofitClient.getClientApi()!!.create(EdcSurcApiService::class.java)
    }

    fun getProvinceApiService() : ProvinceApiService {
        return retrofitClient.getClientApi()!!.create(ProvinceApiService::class.java)
    }

    fun getRegencyApiService(): RegencyApiService{
        return retrofitClient.getClientApi()!!.create(RegencyApiService::class.java)
    }

    fun getDistrictApiService(): DistrictApiService{
        return retrofitClient.getClientApi()!!.create(DistrictApiService::class.java)
    }

    fun getInitialApiService(): InitialApiService{
        return retrofitClient.getClientApi()!!.create(InitialApiService::class.java)
    }

    fun getPriceApiService(): PriceApiService{
        return retrofitClient.getClientApi()!!.create(PriceApiService::class.java)
    }

    fun getStockApiService(): StockApiService{
        return retrofitClient.getClientApi()!!.create(StockApiService::class.java)
    }

    fun getUnitApiService(): UnitApiService{
        return retrofitClient.getClientApi()!!.create(UnitApiService::class.java)
    }

    fun getSignUpApiService() : SignUpApiService {
        return retrofitClient.getClientProvision(BPMConstants.API_MY)!!.create(SignUpApiService::class.java)
    }

    fun getItemDummyApiService() : ItemDummyApiService {
        return retrofitClient.getClientApi()!!.create(ItemDummyApiService::class.java)
    }

    fun getSyncApiService() : SyncApiService {
        return retrofitClient.getClientApi()!!.create(SyncApiService::class.java)
    }

    fun getLicenseApiService() : LicenseApiService {
        return retrofitClient.getClientApi()!!.create(LicenseApiService::class.java)
    }

    fun getGopayApiService() : GopayApiService {
        return retrofitClient.getClientApi()!!.create(GopayApiService::class.java)
    }

    fun getKitchenApiService() : KitchenApiService {
        return retrofitClient.getClientApi()!!.create(KitchenApiService::class.java)
    }

}